app/views/studies/_initialize_expression_form.html.erb

```diff
@@ -102,7 +102,7 @@
 					// auto-select MM Coordinate Matrix if applicable
           fileExt = fileName.split('.').slice(-1)[0];
           if (fileExt === 'mtx') {
```

Comment:
we use `#expression_form_<%= study_file._id %>` more than once -- might be good to pull it into a variable at the top of the script.

Reply:
agreed

Reply:
dcda9f6